Assets/Scripts/LevelScripts/LevelLoader.cs

```diff
@@ -21,7 +21,23 @@ private void Awake()
 
         private void onClick()
```

Comment:
Please use C# naming conventions when writing methods. Also the name of your method doesn't signify anything here. 
When you write a method , it should clearly tell others what it is written for.